reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md

```diff
@@ -44,7 +44,8 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
 
 ## Set up on a Windows computer
 
-1. Install the latest version of PowerShell, see [Installing PowerShell Core on Windows](../../install/installing-powershell-core-on-windows.md#msi).
+1. Install the latest version of PowerShell, see
```

Comment:
This reads like two sentences.  I'd split it up or reword the first half.

Reply:
Fixed